test/broadcast.jl

```diff
@@ -337,3 +337,8 @@ end
 @test broadcast(+, 1.0, (0, -2.0)) == (1.0,-1.0)
 @test broadcast(+, 1.0, (0, -2.0), [1]) == [2.0, 0.0]
 @test broadcast(*, ["Hello"], ", ", ["World"], "!") == ["Hello, World!"]
+
+# Ensure that broadcast doesn't use @inbounds when calling the function
```

Comment:
The tests are run with `—check-bounds=yes`, no?  What's the best way to make sure that this catches regressions?